mapper/orm-batch-jsr352/jberet/pom.xml

```diff
@@ -21,6 +21,7 @@
 
     <properties>
         <java.module.name>org.hibernate.search.batch.jsr352.jberet</java.module.name>
+        <documentation.config.properties.skip>true</documentation.config.properties.skip>
```

Comment:
I'm not sure I want to have to do that...

I agree that, if there's no javadoc to parse, we probably want an error. And then in projects without any javadoc we would set this "skip" property.

But if there is javadoc, and there's simply no settings classes in the module... we can just have the processor end in success without outputting anything?

Reply:
mmm... OK I think we can do it 😄 let me test that

Reply:
Ohh ok... so the trouble is that for this module, we only have impl packages - as a result, the Javadoc is not generated at all (meaning even the `site` folder is not there). So we won't be able to determine if the Javadoc run but didn't generate anything or if the Javadoc didn't run, and that's a build/config issue... 
```xml
<!-- Exclude implementation classes from the javadoc -->
<excludePackageNames>*.impl,*.impl.*</excludePackageNames>
```

Reply:
Ok. Is it the same problem for other modules where you skipped config property collecting? If so, we're good to go.

Reply:
From what it looks, we really need to skip only the jberet module. Skipping orm6/jakarta parents - also makes sense as we are not including any thing from those modules and it would create additional files for ORM-related modules. as we need to add ignore for jberet in these orm6/jakarta modules anyway - adding it on their parents seems to make more sense.